apps/web/src/components/SkillDialog/SkillDialog.tsx

```diff
@@ -6,35 +6,36 @@ import PlusCircleIcon from "@heroicons/react/20/solid/PlusCircleIcon";
 import { Button, Dialog, IconType } from "@gc-digital-talent/ui";
 import { commonMessages } from "@gc-digital-talent/i18n";
 
-import { Scalars, Skill, SkillCategory } from "~/api/generated";
+import { Skill } from "~/api/generated";
 
 import SkillDetails from "./SkillDetails";
 import SkillSelection from "./SkillSelection";
-import { getSkillDialogMessages } from "./utils";
-import { SkillDialogContext } from "./types";
-
-export interface FormValues {
-  category?: SkillCategory | "all" | "";
-  family?: Scalars["ID"];
-  skill?: Scalars["ID"];
-  details?: string;
-}
+import { getSkillDialogMessages, showDetails } from "./utils";
+import { SkillDialogContext, FormValues } from "./types";
 
 const defaultFormValues: FormValues = {
-  category: "all",
-  family: "all",
+  category: "",
+  family: "",
   skill: "",
 };
 
 interface SkillDialogProps {
+  // Al;l available skills
```

Comment:
Typo

Reply:
Oops 🤭 Fixed in [fix typo](https://github.com/GCTC-NTGC/gc-digital-talent/pull/7445/commits/1370e4bcf10a3d6a7664657edd6b83d61b4c994a)